### PR TITLE
Create floating-boxes in InputPage

### PR DESCRIPTION
**Pull request by mnipoiperi**

### CODE REVIEW

File: src/pages/inputPage.js [hunk 1]
```diff
@@ -1,8 +1,112 @@
...
+        {pros.name}  
+      </Text>
+      <TextInput
+        style = {styles.floatingInput}
+        // value
+        onChangeText={onChangeNumber}
+        value = {parseFloat(number)>parseFloat(pros.maxNum)?pros.maxNum:number}
+        placeholder = "0.0"
+        keyboardType = "numeric"
+        // placeholderTextColor = Theme["color-info-300"],
```
Comment (cschenio) [inline src/pages/inputPage.js]: nitpick: remove this

Comment (mnipoiperi) [reply src/pages/inputPage.js]: fixed

File: src/pages/inputPage.js [hunk 2]
```diff
@@ -1,8 +1,112 @@
...
+        maxLength = {5}
+        // reference
+        onSubmitEditing = {pros.refSubmit?() => pros.refSubmit.current.focus():null}
+        returnKeyType={Platform.OS === 'ios' ? 'done' : 'next'}
+        ref = {pros.refMain}
+        // focus style
```
Comment (cschenio) [inline src/pages/inputPage.js]: nitpick: Use full sentence in comment. E.g. `// Focus style.`. Ditto to others.

File: src/pages/inputPage.js [hunk 3]
```diff
@@ -1,8 +1,112 @@
...
+const DateBox = (pros) => {
+  return (
+    <View>
+
+    </View>
+  );
+};
+
+const FloatingBox = (pros) => {
+  const [number, onChangeNumber] = React.useState(null);
```
Comment (cschenio) [inline src/pages/inputPage.js]: suggestion: use `setNumber` instead of `onChangeNumber` to comply with React's convention.

**Review by cschenio — APPROVED** [review]
LGTM, please fix before merge.

**Pull request merged**